### ISSUE TITLE
Focus indicator is partially visible on "Complimentary" button appearing at left side of the window

### ISSUE THREAD

**Issue by melode11** [SeleniumHQ/selenium-ide#1183]
## User Experience
Keyboard user will not able to track the keyboard focus indicator and can not access the control efficiently if focus indicator is partially visible on any control.

## Repro Steps
Open Selenium IDE.
Navigate to  "Complimentary" button appearing at left side of the window
Verify the issue.

## Actual Result
Focus indicator is partially visible on "Complimentary" button appearing at left side of the window
## Expected Result
Focus indicator should be completely visible on "Complimentary" button appearing at left side of the window

## NOTE
Tabstops feature is not working on the page.

## MAS Reference
MAS 2.4.7 - Focus Visible (05A)

## Environment

OS: Windows 10
Selenium IDE Version:  3.17.0
Browser: Chrome
Browser Version: 87.0.4280.141

**Comment by melode11** [comment]
PR merged, close issue.
